### ISSUE TITLE
Test borken, but console show all conditions true

### ISSUE THREAD

**Issue by alisson446** [goldibex/targaryen#105]
The test is breaking, but all conditions than the console show are true. I'm using chai plugin.
It's my rules json:

```json
{
  "rules": {
    "list": {
      "$listId": {
        ".write": "(newData.parent().parent().child('otherList1').child($listId).val() == true)
        	     && (newData.parent().parent().child('otherList2').child($listId).val() == false)"
      }
    }
  }
}
```

And it's is test file: 

```js
const chai = require('chai');
const targaryen = require('targaryen/plugins/chai');
const { expect } = chai;
const { users } = targaryen;

chai.use(targaryen);

describe('list', function() {

  before(function() {
    targaryen.setFirebaseData({});
    targaryen.setFirebaseRules(require('../../database.rules.json'));
  });

  it('', () => {

    const updates = {
      'list/_listId1/id': '_listId1',
      'otherList1/_listId1': true,
      'otherList2/_listId1': false
    };

    expect({ uid: '_uid1' }).can.patch(updates).to.path('/');

  });

});

```
The console: 

![captura de tela de 2016-12-21 16 18 35](https://cloud.githubusercontent.com/assets/13024620/21406462/8eaf37b2-c7aa-11e6-8dc5-837531ee14cb.png)

**Comment by dinoboff** [comment]
It's failing as expected but the info it's showing are wrong:

- only showing one location evaluation.
- mentioning "No .patch rule allowed the operation."

Am I correct?

**Comment by alisson446** [comment]
But why it test should fail?

**Comment by dinoboff** [comment]
@alisson446 

You tests attempt a multi-location update of:
- list/_listId1/id -> '_listId1'.
- otherList1/_listId1 -> true.
- otherList2/_listId1 -> false.

Your rules on allow updating "list/_listId1/id". For "otherList1/_listId1", it should show "otherList1/_listId1:<no rules>".

**Comment by dinoboff** [comment]
ps: AFAIK, permission (writes and read) are denied by default and data is valid by default. So you need one write rule evaluating to true on the path and no validate rule evaluating to false.

**Comment by alisson446** [comment]
I undestand, i'm thought that if doesn't exist rules for a path, it should allow. Thanks.

**Comment by dinoboff** [comment]
@alisson446 The confusing error message should be fixed in in 3.0.0-rc.4.

Thanks.
